core/src/test/scala/org/apache/spark/deploy/SparkSubmitSuite.scala

```diff
@@ -681,7 +681,7 @@ class SparkSubmitSuite
       appArgs.jars should be(Utils.resolveURIs(jars))
       appArgs.files should be(Utils.resolveURIs(files))
       conf.get(JARS) should be(Utils.resolveURIs(jars + ",thejar.jar").split(",").toSeq)
-      conf.get("spark.files") should be(Utils.resolveURIs(files))
+      conf.get(FILES) should be(Utils.resolveURIs(files))
```

Comment:
`FILES` -> `FILES.key` so that it reads it as string like the previous case, or make `Utils.resolveURIs(files)` -> `Utils.resolveURIs(files).split(",").toSeq` to match with the case above.

Reply:
understand